pkg/service/v0/service.go

```diff
@@ -49,12 +55,12 @@ func New(opts ...Option) (s *Service, err error) {
 	indexMapping.DefaultAnalyzer = keyword.Name
 
 	s = &Service{
-		id:     cfg.GRPC.Namespace + "." + "store",
+		id:     strings.Join([]string{cfg.GRPC.Namespace, "store"}, "."),
```

Comment:
I wonder if we should just make the full name configurable. if we want to run multiple intances of ocis-store for different services it might make sense. or we allow configuring labels. can be done at a later point in time ...

Reply:
because we're using the FS as storage, running 2 instances would virtually have the identical path down to the database: `/var/tmp/ocis-store/databases`:

```
/var/tmp/ocis-store/
├── databases
│   └── accounts
│       ├── accounts
│       │   └── foo
│       └── users
│           └── bar
└── index.bleve
    ├── index_meta.json
    └── store
```

I don't imagine that having a different namespace solves an issue. On the other hand, having labels when registering each store would enable us to filter registry results per label and make sure we get the correct store for a given service.

By that argument one could argue we need `/var/tmp/ocis-store/databases` at all, and simply use the service name as "database" name: `/var/tmp/{serviceName}/{tables}` i.e:

```
/var/tmp/ocis-store/
├── ocis-ocs
│   ├── signing-keys
│   │   └── 1b2833dc-12f9-4cd3-ae93-7294deb50050
└── index.bleve
    ├── index_meta.json
    └── store
```

then each service would have its store 🤔 💭